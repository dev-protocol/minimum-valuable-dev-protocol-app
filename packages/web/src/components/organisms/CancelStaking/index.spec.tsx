import React from 'react'
import { render } from '@testing-library/react'
import { CancelStaking } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('@dev/graphql')

describe(`${CancelStaking.name}`, () => {
  test('Snapshot', () => {
    const component = render(<CancelStaking propertyAddress="propertyAddress" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
