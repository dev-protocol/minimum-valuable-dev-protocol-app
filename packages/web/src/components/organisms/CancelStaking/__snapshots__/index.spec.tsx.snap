// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelStaking Snapshot 1`] = `
<body>
  <div>
    <div
      class="dy1yq4-0"
    >
      <p>
        Cancel Staking
      </p>
      <button
        class="ant-btn ant-btn-primary ant-btn-lg"
        type="button"
      >
<<<<<<< HEAD
        <div
          class="ant-space-item"
        >
          <div
            style="display: flex;"
          >
            <div
              style="margin-right: 5px;"
            >
              <button
                class="ant-btn ant-btn-primary ant-btn-lg"
                type="button"
              >
                <span>
                  Cancel
                </span>
              </button>
            </div>
            <button
              class="ant-btn ant-btn-primary ant-btn-lg"
              disabled=""
              type="button"
            >
              <span>
                Withdraw
              </span>
            </button>
            <div />
          </div>
        </div>
      </div>
=======
        <span>
          Cancel
        </span>
      </button>
>>>>>>> upstream/main
    </div>
  </div>
</body>
`;
